<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>80</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/buttons/checkMark_Btn.png</normaloff>:/res/buttons/checkMark_Btn.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>289</y>
       <width>160</width>
       <height>170</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/buttons/upArrow_Btn.png</normaloff>:/res/buttons/upArrow_Btn.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/res/buttons/downArrow_Btn.png</normaloff>:/res/buttons/downArrow_Btn.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>206</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="session_25min">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/session_times/icon_25Min.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="session_45min">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/session_times/icon_45Min.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="session_3hr">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/session_times/icon_3Hour.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="session_custom">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/session_times/icon_customTime.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>206</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>IMG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>IMG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>IMG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>IMG</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>54</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/icon_audioInput.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/icons/graph-icon.png</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="shortPulse_CESsession">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/icon_shortPulse_CESsession.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dutyCycle_CESsession">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/icon_dutyCycle_CESsession.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="left_CESchannel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/icon_LeftCESchannel.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="right_CESchannel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/icon_RightCESchannel.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>130</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/logos/Logo_oasisPro.png</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="col_num_8">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_8.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_7">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_7.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_6">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_6.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_5">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_5.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_4.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_3.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_2.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="col_num_1">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/icons/unLit/colNumbers/icon_1.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>370</y>
       <width>158</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/buttons/power_Btn_unLit.png</normaloff>
       <normalon>:/res/buttons/powerBtn_lit.png</normalon>:/res/buttons/power_Btn_unLit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/icons/INT logo.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/buttons/save_Btn.png</normaloff>:/res/buttons/save_Btn.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>70</y>
       <width>351</width>
       <height>281</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: red solid</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>50</number>
     </property>
     <property name="midLineWidth">
      <number>50</number>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>45</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>45</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Intensity</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>238</red>
         <green>238</green>
         <blue>236</blue>
        </color>
       </brush>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>img</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/icons/Lit/sessionTimes/icon_25Min.png</normaloff>:/res/icons/Lit/sessionTimes/icon_25Min.png</iconset>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignBottom</set>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/icons/Lit/colNumber/icon_5.png</normaloff>:/res/icons/Lit/colNumber/icon_5.png</iconset>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignTrailing|AlignVCenter</set>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/icons/unLit/session_times/icon_45Min.png</normaloff>:/res/icons/unLit/session_times/icon_45Min.png</iconset>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/icons/unLit/colNumbers/icon_7.png</normaloff>:/res/icons/unLit/colNumbers/icon_7.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/icons/icon_audioInput.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
